iFixit/react-commerce
# Add structured data on product page

closes #853 

According to the choices in #972 we inserted the aggregate rating data but not the single reviews.
Brand has been tied to the `vendor` shopify product field. 

cc @ifixitmaxb 

### QA

1. Visit Vercel preview
2. Verify that both `breadcrumbs` and `product` snippets are present on page
3. Verify that they match correctly the InStock/OutOfStock and the New/Refurbished/Used state of the product
